cloud_experts_tutorials/cloud-experts-external-dns.adoc

```diff
@@ -20,25 +20,27 @@ toc::[]
 
 Configuring the xref:../applications/deployments/osd-config-custom-domains-applications.adoc[Custom Domain Operator] requires a wildcard CNAME DNS record in your Amazon Route 53 hosted zone. If you do not want to use a wildcard record, you can use the `External DNS` Operator to create individual entries for routes.
 
-This tutorial guides you through deploying and configuring the External DNS Operator with a custom domain in {product-title} (ROSA).
+Use this tutorial to deploy and configure the `External DNS` Operator with a custom domain in {product-title} (ROSA).
 
 [IMPORTANT]
 ====
-The External DNS Operator does not support IRSA/STS and uses long-lived IAM credentials. This tutorial will be updated once STS is supported.
+The `External DNS` Operator does not support STS using IAM Roles for Service Accounts (IRSA) and uses long-lived Identity Access Management (IAM) credentials instead. This tutorial will be updated when the Operator supports STS.
 ====
 
 [id="cloud-experts-external-dns-prerequisites"]
 == Prerequisites
 
 * A ROSA cluster
-* You have access to the OpenShift CLI (`oc`)
-* You have access to the AWS CLI (`aws`)
+* A user account with `cluster-admin` privileges
+* The OpenShift CLI (`oc`)
+* The Amazon Web Services (AWS) CLI (`aws`)
 * A unique domain, such as *.apps.<company_name>.io
 * An Amazon Route 53 public hosted zone for the above domain
 
-=== Environment setup
+[id="cloud-experts-external-dns-environment-setup"]
+== Setting up your environment
 
-* Prepare the environment variables:
+. Configure the following environment variables, changing the cluster name to suit your cluster:
```

Comment:
I feel that  "changing the cluster name to suit your cluster" could be re-written as it leads me to believe (at first glance) that the name of the cluster hinges on this specific cluster...but the name  is arbitrary? 
I may be other thinking this and I do see that exact phrase is used in another tutorial...so unless something comes easy to you in regards to changing it (it isn't currently to me) I would just leave it. :(

Reply:
> I feel that "changing the cluster name to suit your cluster" could be re-written as it leads me to believe (at first glance) that the name of the cluster hinges on this specific cluster...but the name is arbitrary? I may be over thinking this and I do see that exact phrase is used in another tutorial...so unless something comes easy to you in regards to changing it (it isn't currently to me) I would just leave it. :(

Yeah- it's a bit clumsy. I think I'll change it to 'replacing `CLUSTER_NAME` with the name of your cluster.'